Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:3.1.2
+
+# System prerequisites
+RUN apt-get update \
+ && apt-get -y install build-essential libpq-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+# If you require additional OS dependencies, install them here:
+# RUN apt-get update \
+#  && apt-get -y install imagemagick nodejs \
+#  && rm -rf /var/lib/apt/lists/*
+
+ADD Gemfile /app/
+ADD Gemfile.lock /app/
+WORKDIR /app
+RUN bundle install
```

Comment:
It would be better to do the `bundle install` in frozen mode, similar to my suggestion to the CircleCI config pull request.